### PR TITLE
[TN-29] update readme to include info about .env file for backend

### PR DESCRIPTION
**Pull request by RyanFriedman36**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines. For the guidelines check the README.
- [ ] Tests for the changes have been added (for bug fixes / features). As we discussed this is optional but encouraged.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
  - update readme for .env file in the backend

* **What is the current behavior?** (You can also link to an open issue here)
  -n/a

* **What is the new behavior (if this is a feature change)?**
  -n/a

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
  -no

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 ## Available Scripts
-To start the application, in the backend folder, you can run: 
+In order to start the backend, you must have a .env file in the backend folder with your AWS keys as follows:
```
Comment (csjoblinksreddit) [inline README.md:4]: What do you think about putting this in the project guidelines section?

Comment (RyanFriedman36) [reply README.md:4]: I have no problem with that, I would say that we should then say to reference that section before trying to run. does that sound good?

Comment (csjoblinksreddit) [reply README.md:4]: I agree. I also think I was wrong to put that section so low. Lets put it as the first item in the readme

Comment (RyanFriedman36) [reply README.md:4]: sounds good.

**Pull request closed without merging**